Tests/ImagineEngineTests/TextureManagerTests.swift

```diff
@@ -69,6 +72,30 @@ class TextureManagerTests: XCTestCase {
         XCTAssertEqual(imageLoader.imageNames, ["PrefixTexture.png", "PrefixSecondTexture.png"])
     }
 
+    func testIgnoresErrorWhenErrorModeIsIgnoreWhenLoadinTextureFails() {
+        let texture = Texture(name: "nonExistentTexture")
+        manager.errorMode = .ignore
+
+        _ = manager.load(texture, namePrefix: nil, scale: 1)
+        XCTAssert(errorHandler.didIgnore)
+    }
+
+    func testLogsErrorWhenErrorModeIsLogWhenLoadingTextureFailures() {
+        let texture = Texture(name: "nonExistentTexture")
+        manager.errorMode = .log
+
+        _ = manager.load(texture, namePrefix: nil, scale: 1)
+        XCTAssert(errorHandler.didLog && !errorHandler.didAssert)
```

Comment:
Think it's nicer to use two separate asserts here, one `XCTAssertTrue` and one `XCTAssertFalse`. Makes it much easier to debug failures.

Reply:
Done 👍